src/server.js

```diff
@@ -0,0 +1,29 @@
+require('require-self-ref')
+const {checkWatcher, startNPMWatcher} = require('./watchNPM')
+const redis = require('redis')
+const logger = require('~/src/logging').logger(module)
+const config = require('~/src/config')
+const queue = require('windbreaker-service-util/queue')
+config.load()
+
+;(async () => {
+  const AmqUrl = config.getAmqUrl()
+  const connection = await queue.createConnection({
+    logger,
+    AmqUrl
+  })
+  const channel = await connection.createChannel()
+  await channel.assertQueue(config.getQueueName())
+  const client = redis.createClient(config.getRedisURL())
+  const run = async function () {
+    try {
+      const changes = await startNPMWatcher(channel, client)
+      await setTimeout(checkWatcher(changes, client), 10000)
```

Comment:
If you are using `bluebird` you can use `Promise.delay(timeInMs)` to wait before performing more work